ross/element.py

```diff
@@ -0,0 +1,31 @@
+import pandas as pd
+from abc import ABC
```

Comment:
The `abstractmethod` should be imported from abc:
```python
from abc import ABC, abstractmethod
```

Reply:
Done.

- [element.py](https://github.com/ross-rotordynamics/ross/commit/9bc56eeef8c788cc605ab47ae84fe549eed804e1)